Dockerfile

```diff
@@ -166,6 +166,7 @@ RUN --mount=target=/host \
         -x '*-debugsource-*.rpm' \
         --no-database \
         /host/build/rpms \
+    && echo '%_dbpath %{_sharedstatedir}/rpm' >> /etc/rpm/macros \
```

Comment:
For the lack of a better place to ask a question about something mentioned in the commit message (and for my own understanding):

Why should `/usr`  be a symlink to the sysroot directory?

@bcressey kindly replied offline:

> If it is anything other than a symlink, then packages from Bottlerocket could clash with packages from Fedora at build time.
>By keeping the two roots separate until the last minute, we avoid some conflicts and chances of pollution. The symlink lets us have normal paths at runtime.